jvm/src/main/scala/uk/gov/homeoffice/drt/db/dao/StaffShiftsDao.scala

```diff
@@ -5,17 +5,24 @@ import slick.lifted.TableQuery
 import uk.gov.homeoffice.drt.db.CentralDatabase
 import uk.gov.homeoffice.drt.db.tables.{StaffShiftRow, StaffShiftsTable}
 
+import java.sql.Date
 import scala.concurrent.Future
 
 trait IStaffShiftsDao {
```

Comment:
The DAO functions should take `Shift`s rather than `ShiftRow`s. The `ShiftRow` is the implementation detail of how we store shifts, so it should be hidden from the DAO caller.
The DAO should convert (or serialise) Shifts to ShiftRows and visa versa